src/styles/main.scss

```diff
@@ -1,2 +1,65 @@
-@import "blocks/calendar";
-@import "blocks/cell.scss"
+$blockSize: 100px;
+$weekDays:
+  mon 0,
+  tue 101px,
+  wed 202px,
+  thu 303px,
+  fri 404px,
+  sat 505px,
+  sun 606px;
+
+.calendar {
+  position: absolute;
+  display: flex;
+  flex-wrap: wrap;
+
+  padding: 100px;
+  width: 710px;
+  gap: 1px;
+
+  &--month-length {
+    @for $day from 28 through 30 {
+      &--#{$day} .calendar__cell:nth-child(n+#{$day + 1}) {
+        display: none;
+      }
+    }
+  }
+
+  &--start-day {
+    @each $day, $cell in $weekDays {
+      &--#{$day} .calendar__cell:first-child {
+        margin-left: $cell;
+      }
+    }
+  }
+
```

Comment:
If you change the block size, you'll have problems with markup. That's why it's better to do this
![image](https://github.com/mate-academy/layout_calendar/assets/122356496/7537f70a-bda0-45f8-919f-a9fb299e8348)
![image](https://github.com/mate-academy/layout_calendar/assets/122356496/c71656e1-dfc6-4b8d-82f8-6524b0999765)

Reply:
```
$weekDays:
  mon 0,
  tue 1,
  wed 2,
  thu 3,
  fri 4,
  sat 5,
  sun 6;
$blockSize: 100px;
$gap: 1px

.calendar {
  position: absolute;
  display: flex;
  flex-wrap: wrap;

  padding: $blockSize;
  width: 710px;
  gap: $gap;

  &--month-length {
    @for $day from 28 through 30 {
      &--#{$day} > .cell:nth-child(n+#{$day + 1}) {
        display: none;
      }
    }
  }

  &--start-day {
    @each $day, $cell in $weekDays {
      &--#{$day} > .cell:first-child {
        margin-left: $cell * $BlockSize + $gap * 7;
      }
    }
  }
}
```